### ISSUE TITLE
The examples in offical website are too simple

### ISSUE THREAD

**Issue by microkof** [pklauzinski/jscroll#125]
This project is very good, but I suggest to detail the examples in offical website.

The examples in offical website are too simple, and there is no example in github repository.

**Comment by pklauzinski** [comment]
@microkof do you have any examples of your own you'd like to contribute?

**Comment by microkof** [comment]
@pklauzinski sorry, I posted this issue because I don't know well about this project. So I suggest you add more examples, the code is too simple, I can't understand. I am a beginner, so can you add complete code?

please no abbreviated, more detailed.thank you!

**Comment by gvlasov** [comment]
@pklauzinski For me, the problem with the examples is that they don't tell you at all what do you need to do to get the updated content. I'd expect there to be a way to define a URL parameterized with the scroll position to asynchronously get the next portion of data, and a way to render that data after receiving it. But there is simply nothing like that in the documentation.

**Comment by enricopalazzo** [comment]
I agree that a full example files would be good for newbies like me, that would like to see how the HTML part is done and also use it as system/user error check. 

I've been trying unsuccessfully to make this work using the abbreviated HTML sample in the docs, so a full example could help me check what am I doing wrong, or if there's some other issue.
